spec/services/invite_provider_user_spec.rb

```diff
@@ -31,6 +31,13 @@
       it 'a provider user is created' do
         expect(ProviderUser.find_by_email_address('test+invite_provider_user@example.com')).not_to be_nil
       end
+
+      it 'queues an email' do
+        message_delivery = instance_double(ActionMailer::MessageDelivery, deliver_later: nil)
+        allow(ProviderMailer).to receive(:account_created).and_return(message_delivery)
+        InviteProviderUser.new(provider_user: new_provider_user_from_form).save_and_invite! rescue nil
+        expect(ProviderMailer).to have_received(:account_created)
```

Comment:
Perhaps instead of this we can add a [test in the system spec](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/spec/system/support_interface/managing_provider_users_spec) that confirms that the email is sent? This feels too much coupled to the internals.

Reply:
Have added the assert to the system spec - it does require the feature flag to make this work.